src/software/ai/passing/cost_function.cpp

```diff
@@ -67,6 +67,9 @@ double Passing::ratePass(const World& world, const Passing::Pass& pass,
 double Passing::ratePassShootScore(const Field& field, const Team& enemy_team,
                                    const Passing::Pass& pass)
 {
+    // TODO: You don't even use this first parameter, but stuff is hardcoded below
+    double ideal_shoot_angle_degrees =
+        Util::DynamicParameters::Passing::ideal_min_shoot_angle_degrees.value();
```

Comment:
this is a bad merge, please update with the evaluation.cpp on master

Reply:
I don't think I understand what you mean by that.  Could you elaborate on what you want me to do?